db/models.py

```diff
@@ -50,3 +60,78 @@
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(null=True, blank=True)
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE)
+
+    class Meta:
+        ordering = ["-created_at"]
+
+    def __str__(self) -> str:
+        return f"{self.created_at}"
+
+
+class Ticket(models.Model):
+    movie_session = models.ForeignKey(
+        to=MovieSession,
+        on_delete=models.CASCADE,
+        related_name="tickets"
+    )
+    order = models.ForeignKey(
+        to=Order,
+        on_delete=models.CASCADE,
+        related_name="tickets"
+    )
+    row = models.IntegerField()
+    seat = models.IntegerField()
+
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(
+                fields=["movie_session", "row", "seat"],
+                name="unique_session_row_and_seat"
+            )
+        ]
+
+    def clean(self) -> None:
+        if not (isinstance(self.row, int) and isinstance(self.seat, int)):
+            raise ValidationError(
+                "`rows` and `seats` must be integers numbers"
+            )
+        rows_in_hall = self.movie_session.cinema_hall.rows
+        seats_in_row = self.movie_session.cinema_hall.seats_in_row
+        if not 1 <= self.row <= rows_in_hall:
+            raise ValidationError(
+                {
+                    "row": [
+                        f"row number must be in available range: "
+                        f"(1, rows): (1, {rows_in_hall})"
+                    ]
+                }
+            )
+        if not 1 <= self.seat <= seats_in_row:
+            raise ValidationError(
+                {
+                    "seat": [
+                        f"seat number must be in available range: "
+                        f"(1, seats_in_row): (1, {seats_in_row})"
+                    ]
+                }
+            )
```

Comment:
The `clean` method in the `Ticket` model ensures that `row` and `seat` are within valid ranges. This is a good practice for maintaining data integrity.